EddystoneBeacon.cpp

```diff
@@ -7,16 +7,17 @@
 #define FLAGS_TLM 0x20
 
 static const char* EDDYSTONE_URL_BEACON_PREFIX_SUBSTITUTIONS[] = {
-  "http://www.",
-  "https://www.",
+  "http://www.",//for http://www.
+  "https://www", //for .
   "http://",
   "https://",
-  "urn:uuid:"
+  "urn:uuid:",
+  "http//localhost:" //additional url prefix for connecting to localhost.
 };
 
 static const char* EDDYSTONE_URL_BEACON_SUFFIX_SUBSTITUTIONS[] = {
-  ".com/",
-  ".org/",
+  ".c",  //com
+  ".o",  //org
```

Comment:
These changes are not up to spec, see: https://github.com/google/eddystone/tree/master/eddystone-url#eddystone-url-http-url-encoding